File: 프로그래머스/unrated/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.py
def solution(myString):
    # myString을 'x'를 기준으로 잘라 배열로 만든다.
    # 리스트 컴프리헨션으로 빈 문자열이 아닌 것만 남겨 정렬한다.
    return sorted([letters for letters in myString.split('x') if letters != ''])

File: 프로그래머스/unrated/181866. 문자열 잘라서 정렬하기/문자열 잘라서 정렬하기.js
function solution(myString) {
    // myString을 'x'를 기준으로 잘라 배열을 만든 후, 빈 문자열이 아닌 것만 남기고, 정렬한다.
    return myString.split('x').filter(el => el != '').sort()
}